src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,13 @@
+@each $number, $day in $name_days {
+  .calendar--start-day-#{$day} :first-child {
+    margin-left: calc(#{$number} * #{$space});
```

Comment:
1. I think here you just need `($field-width + $calendar-gap) * $ number`
2. it's not necessary to write calc using scss 😉

Reply:
$space: calc(#{$field-width} + #{$calendar-gap});
without calc it won't work